megamek/src/megamek/client/ui/swing/util/KeyCommandBind.java

```diff
@@ -122,7 +122,8 @@ public enum KeyCommandBind {
     SENSOR_RANGE(true, "sensorRange", VK_C),
     UNDO_SINGLE_STEP("undoSingleStep", VK_BACK_SPACE, CTRL_DOWN_MASK),
     FORCE_DISPLAY(true, "toggleForceDisplay", VK_F, CTRL_DOWN_MASK),
-    EXTEND_TURN_TIMER("extendTurnTimer", VK_F4, CTRL_DOWN_MASK);
+    EXTEND_TURN_TIMER("extendTurnTimer", VK_F4, CTRL_DOWN_MASK),
```

Comment:
This gives me the willies but <CTRL>-<F4> _should_ be far enough from <ALT>-<F4> that people won't hit the wrong combo by accident... right?